src/ng/compile.js

```diff
@@ -3052,8 +3052,8 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
 
       $compileNode.empty();
 
-      $templateRequest(templateUrl)
-        .then(function(content) {
+      $templateRequest(templateUrl).
```

Comment:
Knee-jerk reaction, sorry 😞 I'll amend.

<sub>(Although, there are some occurrences and I myself have sneaked in a [couple of those][1] lately 😛)</sub>

[1]: https://github.com/angular/angular.js/blob/823295fee058f169523e84f94be13962a9513d3d/src/ngRoute/route.js#L679-L685